logic.js

```diff
@@ -0,0 +1,96 @@
+const Brainer = function(){
+
+    ///Frog creation:
+
+
+    const frogStyles = []
+
+    const creatFrogStyleArry = () => {
+        frogStyles.length = 0
```

Comment:
There is no need to assign the length value. An array length is depending on pushing or splicing items from it.